ruby-challenges.rb

```diff
@@ -32,3 +63,31 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# Pseudocode: 1.) Create a method called is_palindrome
+# 2.) is_palindrome will take in a string argument
+# 3.) use .downcase! to change all the characters to a lower case
+# 4.) use and if statement and an equality operator to check if the reversed string = string
+# 5.) return a string interpolated statment if the argument is palindrome or not.
+
+def is_palindrome (string)
+    string.downcase!
+    if string == string.reverse
+      "#{string}, is a palindrome"
+    else
+        "#{string}, is not a palindrome"
+    end
+end
+
```

Comment:
Nice job! I see that you are trying to handle edge cases here with `.downcase`. However, the `!` is making the change permanent, which is why it is still lowercase in the string that is returned instead of the original string.